### PR TITLE
Gardener gcp job

### PR DESCRIPTION
**Pull request by clebs**
**Description**

Changes proposed in this pull request:

- Add new gardener GCP periodic job to test vanilla Kyma.
- Rename azure job to diferentiate with the new one.

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/7068

### CODE REVIEW

File: development/tools/jobs/kyma/kyma_gardener_integration_test.go [hunk 1]
```diff
@@ -27,8 +27,32 @@ func TestKymaGardenerAzureIntegrationJobPeriodics(t *testing.T) {
...
+func TestKymaGardenerGCPIntegrationJobPeriodics(t *testing.T) {
+	// WHEN
+	jobConfig, err := tester.ReadJobConfig("./../../../../prow/jobs/kyma/kyma-integration-gardener-gcp.yaml")
+	// THEN
+	require.NoError(t, err)
+
+	periodics := jobConfig.Periodics
+	assert.Len(t, periodics, 1)
+
+	jobName := "kyma-gardener-azure-integration"
```
Comment (suleymanakbas91) [inline development/tools/jobs/kyma/kyma_gardener_integration_test.go]: Should be `kyma-gardener-gcp-integration`

File: prow/jobs/kyma/kyma-integration-gardener-gcp.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Kubectl 1.16
+base_image: &base_image
+  image: eu.gcr.io/kyma-project/test-infra/kyma-cluster-infra:v20200206-22eb97a4
+
+periodics:
+- name: kyma-gardener-gcp-integration
```
Comment (suleymanakbas91) [inline prow/jobs/kyma/kyma-integration-gardener-gcp.yaml]: I think the naming should be consistent everywhere. File name is `kyma-integration-gardener-gcp` and job name is `kyma-gardener-gcp-integration`

File: development/tools/jobs/kyma/kyma_gardener_integration_test.go (hunk 1 above)
Comment (clebs) [reply development/tools/jobs/kyma/kyma_gardener_integration_test.go]: Yes, I actually forgot to commit this change on the test...
SHould be fixed now.

File: prow/scripts/cluster-integration/kyma-integration-gardener-gcp.sh [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+RANDOM_NAME_SUFFIX=$(LC_ALL=C tr -dc 'a-z0-9' < /dev/urandom | head -c6)
+readonly COMMON_NAME_PREFIX="grdnr"
+COMMON_NAME=$(echo "${COMMON_NAME_PREFIX}-${RANDOM_NAME_SUFFIX}" | tr "[:upper:]" "[:lower:]")
+
+### Cluster name must be less than 20 characters!
+export CLUSTER_NAME="${COMMON_NAME}"
+
+# Local variables
+# DNS_SUBDOMAIN="${COMMON_NAME}"
```
Comment (suleymanakbas91) [inline prow/scripts/cluster-integration/kyma-integration-gardener-gcp.sh]: You can delete this

File: prow/jobs/kyma/kyma-integration-gardener-gcp.yaml (hunk 2 above)
Comment (clebs) [reply prow/jobs/kyma/kyma-integration-gardener-gcp.yaml]: Fixed thanks!

File: prow/scripts/cluster-integration/kyma-integration-gardener-gcp.sh (hunk 3 above)
Comment (clebs) [reply prow/scripts/cluster-integration/kyma-integration-gardener-gcp.sh]: Fixed thanks

**Pull request merged**